setup.py

```diff
@@ -44,6 +44,7 @@
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
+    python_requires=">=3.7",
```

Comment:
I already merged a python_requires line in https://github.com/intel/cve-bin-tool/commit/37b99a9b242392d7d1f6a889b9221bb5fcfa5e44 so you can probably leave this change out of the PR.

Reply:
Ok